xformers/csrc/attention/cuda/fmha/kernel_backward.h

```diff
@@ -1430,6 +1432,7 @@ struct AttentionBackwardKernel {
     cutlass::MatrixCoord no_offset{0, 0};
     accum_t scale = p.scale;
     int16_t thread_id = 32 * warp_id + lane_id;
+    const accum_t alibi_base = powf(static_cast<float>(pow(2.0, -pow(2.0, -(log2(static_cast<double>(p.num_heads)) - 3.0)))), static_cast<float>(blockIdx.y + 1));
```

Comment:
How about:
```suggestion
    const accum_t alibi_base = powf(static_cast<float>(powf(2.f, -powf(2.f, 3.f - log2f(static_cast<float>(p.num_heads))))), static_cast<float>(blockIdx.y + 1));
```
So that we compile to CUDA intrinsics. Or do you think double precision really matters in `num_heads`?

Reply:
I will check once again after all fixes. At some point double here seemed to improve precision

Reply:
Double precision power is quite expensive... I also noticed that we can simplify `static_cast<float>(powf` -> `powf`.

Reply:
Ok, after all changes it does not matter here